aio/aio-proxy/aio_proxy/routes.py

```diff
@@ -25,6 +26,8 @@
     sentry_sdk.init(
         dsn=DSN_SENTRY,
         integrations=[AioHttpIntegration(transaction_style="method_and_path_pattern")],
+        level=logging.INFO,  # Capture info and above as breadcrumbs
+        event_level=logging.ERROR,  # Send errors as events
         # Log 10% of transactions for performance monitoring
```

Comment:
Was this for dev purpose ? Or do you really want to change the logging level ?

Reply:
It's the default confit for Sentry. It does not change anything. I might want to change the event level to `info` to get cache info.